chess/db/package.json

```diff
@@ -6,7 +6,7 @@
   "types": "build/index.d.ts",
   "type": "module",
   "scripts": {
-    "clean": "rm -rf tsconfig.tsbuildinfo & rm -rf build",
```

Comment:
Is there a reason you put these in the `db/package.json` instead of the root `package.json`?

Reply:
No, not really, I'm just used to localize the scripts to where they are ran, and it made more sense to have it there when the script was `rm` -rf tsconfig.tsbuildinfo & rm -rf build`, but now, that we are using `wipe.sh`, it does make sense to run it from the root.

However, I would suggest to also run `clean` when running `build`, since that is where it is of most use, because if not, it will fail silently (fail as in not produce new output), which was the reason why I wanted to add this in the first case. And in this case, it is a bit awkward to have the `clean` script in the parent `package.json` and the `build` script in `db/package.json`.

I guess it boils down to if you expect users to run scripts directly from the `db` package, or always operate from the root.

Reply:
> However, I would suggest to also run clean when running build

it should never be required to run `clean` before `build`. If your code changes, it will automatically know to rebuild (that's what `tsconfig.tsbuildinfo` is for). Usually the only time typescript fails to detect changes are cases related to bumping dependencies that include special tools (which is why the `wipe.sh` command also wipes `node_modules`

Reply:
ok, understood. In that case there is something else at play here, I would think. I need to investigate further.